packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.jsx

```diff
@@ -0,0 +1,64 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const TurbosInfo = ({ min_stack = 1, className, currency, has_stop_loss, max_stack = 100 }) => {
+    const min_stake_text = (
+        <Text
+            as='p'
+            line_height='s'
+            size='xxxs'
+            className={classNames({
+                [`${className}-tooltip-text`]: className,
+            })}
+        >
+            <Localize
+                i18n_default_text='Min. stake <0/>'
+                components={[<Money key={0} amount={min_stack} currency={currency} show_currency />]}
+            />
+        </Text>
+    );
+
+    const max_stake_text = (
```

Comment:
I think it will be possible to reuse min_stake_text for max_stake_text, so I'm going to improve it today

Reply:
Because the only difference between them is default text and amount